src/main/java/com/mraof/minestuck/tileentity/redstone/WirelessRedstoneReceiverTileEntity.java

```diff
@@ -55,21 +55,30 @@ public void setLastTransmitterBlockPos(BlockPos lastTransmitterPosIn)
 	 */
 	public void renewFromLastTransmitter()
 	{
-		if(level != null && lastTransmitterBlockPos != null && !level.isClientSide && level.isAreaLoaded(lastTransmitterBlockPos, 1))
+		if(level != null && !level.isClientSide)
 		{
-			TileEntity tileEntity = level.getBlockEntity(lastTransmitterBlockPos);
-			if(tileEntity instanceof WirelessRedstoneTransmitterTileEntity)
+			BlockState state = getBlockState();
+			BlockState newState = WirelessRedstoneReceiverBlock.setPower(state, 0);
```

Comment:
This is only the new state if it can't find the transmitter. Perhaps call it `unpoweredState`?